readthedocs/oauth/migrations/0014_remove_field_from_remote_repository_model.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.16 on 2020-10-05 06:18
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('oauth', '0013_data_migration_for_remote_relation_model'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='remoterepository',
+            name='account',
+        ),
```

Comment:
Now that we are moving the `SocialAccount` to the `RemoteRelation`, shouldn't we have a `provider` or similar in the `RemoteRepository` to know to which service this repository belongs to?

Reply:
Yes! Great idea :100:  It will be better to add a ``provider`` field so we don't have to do an extra query to know the provider name or if the user is deleted or disconnected we don't lose that data.

Some thought on migration of this data here https://github.com/readthedocs/readthedocs.org/pull/7536#discussion_r501799277